src/components/Create/components/pages/NftRewards/NftRewardsPage.tsx

```diff
@@ -94,6 +119,36 @@ export const NftRewardsPage = () => {
             <CreateCollapse>
               <CreateCollapse.Panel
                 key={1}
+                header={<OptionalHeader header={t`On-chain Governance`} />}
+                hideDivider
+              >
+                <Form.Item name="onChainGovernance">
+                  <Radio.Group className="flex flex-col gap-5">
+                    <RadioItem
+                      value={JB721GovernanceType.NONE}
+                      title={
+                        <>
+                          <Trans>No on-chain governance capabilities</Trans>{' '}
+                          <CreateBadge.Default />
+                        </>
+                      }
+                      description={t`Your project's NFTs will not have any governance capabilities.`}
```

Comment:
I think these should all be `xs` too.

Reply:
These were based off @strath-m design:
![image](https://user-images.githubusercontent.com/104132113/214976765-4e3fb927-a926-4aa7-8901-6d2b921ec928.png)


For comparison, this is what xs looks like:
![image](https://user-images.githubusercontent.com/104132113/214976873-d28d2517-7bf6-4c33-b7de-4bd5ea6d96e7.png)